src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
@srivatsn But what is the state of the user's session at that point?  It's really going to depend on what that process was doing.  Is there a way to tag a process with "I'm not doing anything mission critical, ignore my crash" vs. "The user's session will be totally messed up if the process dies?"  That sounds like what we should be gating on, really.